publish-package/action.yml

```
@@ -0,0 +1,48 @@
+# Copyright 2022-2023 Universität Tübingen, DKFZ and EMBL
+# for the German Human Genome-Phenome Archive (GHGA)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: PyPI Publish
+description: Tasks that are executed in GHGA PyPI publish pipeline 
+
+inputs:
+  test_pypi_api_token:
+    description: "Token required to publish the package to test pypi"
+    required: true
+  pypi_api_token:
+    description: "Token required to publish the package to test pypi"
+    required: true
+  package_dir: 
+    description: "Custom target package dists directory"
+    default: "./artifact"
+  
+
+runs:
+    using: composite
+
+    steps:
+      - name: Publish distribution package to PyPI (test)
+        uses: pypa/gh-action-pypi-publish@release/v1
+        # if: ${{ !env.ACT }}
+        with:
+          password: ${{ inputs.test_pypi_api_token }}
+          repository-url: https://test.pypi.org/legacy/
+          packages-dir: ${{ inputs.package_dir }}
+
+      - name: Publish distribution package to PyPI (production)
+        uses: pypa/gh-action-pypi-publish@release/v1
+        # if: ${{ !env.ACT }}
```

```suggestion
        if: ${{ !env.ACT }}
```
Never publish to official pypi with act. Is that actually an issue with the test workflow? Is the dummy-package published? We should check that and maybe add an input var that skips this step during testing.